### PR TITLE
ImGuiIntegration: make ImFontAtlas texture accessible

### LINKED ISSUE #60: ImFontAtlas::TexID remains unset after a call to ImGuiIntegration::Context::relayout
`ImGuiIntegration::Context::relayout()` creates the `GL::Texture2D` for the font atlas, but does not set the `TexID` in the `ImFontAtlas` object.

I may have just missed something, but I couldn't find any way to access the imgui font atlas texture.

Simple fix tho!

https://github.com/mosra/magnum-integration/blob/43e86c2a1e83caf9b4da42e00c8e74ccd3253d89/src/Magnum/ImGuiIntegration/Context.cpp#L228

Around there this line should fix it:

`io.Fonts->SetTexID(reinterpret_cast<ImTextureID>(&_texture));`
or
`io.Fonts->SetTexID(&_texture);`

I am unsure if the reinterpret cast is needed, I would expect it, because `ImTextureID` is void*.

It seems like `static_cast<ImTextureID>()` is used elsewhere, so idk.

### PR DESCRIPTION
**Pull request by bowling-allie**
Fixes #60

### CODE REVIEW

File: src/Magnum/ImGuiIntegration/Context.h [hunk 1]
```diff
@@ -423,6 +423,13 @@ class MAGNUM_IMGUIINTEGRATION_EXPORT Context {
          */
         void drawFrame();
 
+        /**
+         * @brief Get the ImFontAtlas texture
+         *
+         * @return Reference to the @cpp GL::Texture2D @ce in @cpp ImFontAtlas @ce
+         */
+        GL::Texture2D& getAtlasTexture();
```
[the remark this answers is not in the document]
Comment (bowling-allie) [reply src/Magnum/ImGuiIntegration/Context.h]: yes, my bad

**Comment by mosra** [comment]
Merged as b3c65b91b3d13591244dc8be75eb5f878ec70d68, thank you! :)

**Comment by bowling-allie** [comment]
btw i have been testing with the ternary statement simplified, and everything works

**Comment by Squareys** [comment]
There is a tiny bug in here:

When the Context is moved, the pointer to `_texture` is invalidated so that the ImGui font TexID points to invalid memory. The TexID needs to be updated in `Context(Context&&)`.

**Pull request closed without merging**